.github/workflows/ci.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
@stefanbuck looks like this fixes the test issue we've been having. Not sure why it didn't do anything in #1035 but it worked here.